go.mod

```diff
@@ -424,3 +424,5 @@ replace (
 //
 // Then, run `make modtidy` in this repository.
 // This ensures that go.mod and go.sum are up-to-date.
+
+replace github.com/dapr/components-contrib => github.com/DeepanshuA/components-contrib v1.6.0-rc.2.0.20230327134753-c1df8bfd5f5e
```

Comment:
Reviewers: I will remove it and refer contrib with interface changes, before this PR gets ready to be merged.